examples/CMakeLists.txt

```diff
@@ -34,20 +34,33 @@ option(CONNEXTDDS_BUILD_WEB_INTEGRATION_SERVICE_EXAMPLES
     OFF
 )
 
+option(CONNEXTDDS_BUILD_CONNEXT_SECURE_EXAMPLES
+    "Build Connext Secure examples"
+    OFF
```

Comment:
@lulivi I'm not sure this needs to be disabled or not. Can you advise?

Reply:
I guess it depends on whether we can build the new examples with the current GitHub pipeline. I'd assume it would not be too complicated to do so, with the exception of running the `setup.py` python script. This is a different step that in other examples, so it may not be straight-forward. Let's wait for @lulivi's opinion on it.

Reply:
We currently don't support the `setup.py` Python script as @colmenero mentioned. So I would leave this OFF for now. We can open an issue in order to configure a step of running setup before the examples.
Also I would suggest a different name for the script since it is the common name for any Python package entry point (https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#setup-py) and could cause confusion and possible problems with some tools.

Reply:
Renaming to `setup_security.py`